core/src/test/java/org/opensearch/sql/expression/operator/arthmetic/ArithmeticFunctionTest.java

```diff
@@ -113,7 +113,7 @@ public void mod(ExprValue op1, ExprValue op2) {
     assertEquals(String.format("mod(%s, %s)", op1.toString(), op2.toString()),
             expression.toString());
 
-    expression = DSL.mod(literal(op1), literal(new ExprShortValue(0)));
+    expression = DSL.mod(literal(op1), literal(new ExprByteValue(0)));
```

Comment:
why not keep some of these as short, some as byte, and some as other types (just to add coverage)

Reply:
What is happening here is that the function is processed by the widest data type, so having this value be byte (the smallest data type) means that it will always be processed through the arithmetic function of the data type of op1, which goes through all of them.

Because before I had the value be a short, any byte value that went in was processed as a short (as short is the wider type) meaning that I didn't see that dividing by 0 didn't work

I did think of an alternative solution that involves taking out the 0 denominator testing from here and adding an addition test which explicitly tests every single data type in a cleaner way, although I found it unnecessary because of all the addition code.